package.json

```diff
@@ -15,12 +15,17 @@
     "lint:fix": "npm run lint -- --fix"
   },
   "dependencies": {
+    "@react-native-community/masked-view": "^0.1.10",
```

Comment:
This dependency should be removed, it's not used plus the docs mentions "Note: React Native MaskedView is not currently supported by Expo unless you "eject"."

Reply:
oh just checked, in https://reactnavigation.org/docs/getting-started, @react-native-community/masked-view is a dependency cos of the safe area view